4_arm.ipynb

```diff
@@ -581,11 +1058,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "レゴブロックを取るのに失敗した場合、以下のコマンドでレゴブロックを消して、再度出現させると元の取りやすい位置に戻ります。再挑戦してみましょう。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "If you fail to pick the Lego block, delete the Lego block with the following command and make it reappear to the original easy-to-take position:"
```

Comment:
"If you fail to pick up the Lego block, then delete the Lego block with the following command and make it reappear at the original position from where it is easy to pick it up from:"